06-express-and-layered-architecture/src/index.ts

```diff
@@ -0,0 +1,14 @@
+import express from "express";
+import cartController from "./cart/cart.controller";
+import productController from "./product/product.controller";
+
+const app = express();
+const port = 8000;
+
+app.use(express.json());
+app.use(cartController);
```

Comment:
It's a good practice to differentiate the two Router middlewares in the main entry file, in my opinion. So it would be nice to see the path on which each of these controllers operate, i.e.: `app.use('/api/profile/cart', cartController)`